frontend/src/components/ProfileView/ProfileView.module.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+}

```

Comment:
again, we should refrain from using element selectors like this body tag in our css stylesheet because it will affect the styles of ALL body tags, not just your own ProfileView component.